app/controllers/api/v1/visualization/explore_controller.rb

```diff
@@ -83,8 +84,10 @@ def show
             annotationList: AnnotationVizService.get_study_annotation_options(@study, current_api_user),
             clusterGroupNames: ClusterVizService.load_cluster_group_options(@study),
             spatialGroups: spatial_group_options,
+            imageFiles: image_options,
             clusterPointAlpha: @study.default_cluster_point_alpha,
-            colorProfile: @study.default_color_profile
+            colorProfile: @study.default_color_profile,
+            bucket_id: @study.bucket_id
```

Comment:
Blocker: let's use consistent case in API response objects, or add a code comment on why this is an exception.

```suggestion
            bucketId: @study.bucket_id
```